__tests__/html/transcript.upDownArrow.html

```diff
@@ -0,0 +1,228 @@
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} type="checkbox" value={'<input type="checkbox">'} />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} type="file" />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} type="image" value={'<input type="image">'} />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} type="radio" value={'<input type="radio">'} />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} type="reset" value={'<input type="reset">'} />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} type="submit" value={'<input type="submit">'} />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} placeholder={'<input type="email">'} type="email" />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} placeholder={'<input type="password">'} type="password" />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} placeholder={'<input type="search">'} type="search" />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} placeholder={'<input type="tel">'} type="tel" />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} placeholder={'<input type="text">'} type="text" />
+                  </article>
+                  <article>
+                    <input onChange={NULL_FN} placeholder={'<input type="url">'} type="url" />
+                  </article>
+                  <article>
+                    <textarea placeholder={'<textarea>'} />
+                  </article>
+                  <article>
+                    <div
+                      className="content-editable-placeholder"
+                      contentEditable={true}
+                      suppressContentEditableWarning={true}
+                    />
+                  </article>
+                  <article>
+                    <div tabIndex={0}>{'<div tabIndex={0}>'}</div>
+                  </article>
+                  <article>
+                    <input
+                      autoComplete="off"
+                      onChange={NULL_FN}
+                      placeholder={'<input autoComplete="off" type="text">'}
+                      type="text"
+                    />
+                  </article>
+                </section>
+                <section className="elements-should-not-scroll">
+                  <h1>Elements here should NOT scroll on arrow up/down</h1>
+                  <article>
```

Comment:
non-blocking comment: shouldn't `<article>` have labels inside?

Reply:
No, labels are not needed. This is HTML for testing and not designed for accessibility.